renderer/index.js

```diff
@@ -153,6 +153,11 @@ function cleanUpConfig () {
       delete ts.posterURL
       ts.posterFileName = infoHash + extension
     }
+
+    // Migration: add per-file selections
+    if (!ts.selections) {
+      ts.selections = ts.files.map((x) => true)
```

Comment:
I guess `fileModTimes` currently lives outside of `files` too. So nevermind, I think this is fine for now.